src/Configuration/AccountManager.cs

```diff
@@ -9,42 +10,64 @@ internal static class AccountManager
     {
         public static List<Account> Accounts;
 
-        private static void Load(string serverName)
+        public static string[] GetAccountNames(string serverName)
```

Comment:
I would prefer an `IReadOnlyCollection<Account>` as return value

Reply:
that would work.

Reply:
changed